open_feature/hooks/hook_support.py

```diff
@@ -0,0 +1,128 @@
+import logging
+import typing
+from functools import reduce
+
+from open_feature.evaluation_context.evaluation_context import EvaluationContext
+from open_feature.flag_evaluation.flag_evaluation_details import FlagEvaluationDetails
+from open_feature.flag_evaluation.flag_type import FlagType
+from open_feature.hooks.hook import Hook
+from open_feature.hooks.hook_context import HookContext
+from open_feature.hooks.hook_type import HookType
+
+
+def error_hooks(
+    flag_type: FlagType,
+    hook_context: HookContext,
+    exception: Exception,
+    hooks: typing.List[Hook],
+    hints: dict,
+):
+    """
+
+    :param flag_type:
```

Comment:
We should either complete these docstrings or remove them

Reply:
Docstrings updated